services/eventhistory/README.md

```diff
@@ -0,0 +1,27 @@
+# Eventhistory Service
+
+The `eventhistory` consumes all events from the configured event system like NATS, stores them and allows to retrieve them via an eventid.
+
+## Prerequisites
+
+Running the eventhistory service without an event sytem like NATS is not possible.
+
+## Consuming
+
+The `eventhistory` services consumes all events from the configured event sytem.
+
+## Storing
+
+The `eventhistory` stores each consumed event in the configured store. Possible stores are:
+  - `mem`: Basic inmemory store and the default.
+  - `ocmem`: Advanced inmemory store allowing max size.
+  - `redis`: Stores data in a configured redis cluster.
+  - `etcd`: Stores data in a configured etcd cluster.
+  - `nats-js`: Stores data using nats.
+  - `noop`: Stores nothing. Useful for testing. Not recommended in productive enviroments.
+
+Events stay in the store for 2 weeks by default. Use `EVENTHISTORY_RECORD_EXPIRY` to adjust this value.
+
+## Retrieving
+
+Other services can call the `eventhistory` service via a grpc call to retrieve events. The request must contain the eventid that should be retrieved.
```

Comment:
> The request must contain the eventid that should be retrieved.

How does one know a eventID?

Reply:
One cannot. As said it is only other services that can call eventhistory service. Other service will automatically get the id when consuming events.